### PR TITLE
feat(preset, conventionalcommits): Add handling of issue prefixes

### PR DESCRIPTION
**Pull request by tommywo**
fixes conventional-changelog/conventional-changelog-config-spec#19

### CODE REVIEW

File: packages/conventional-changelog-conventionalcommits/test/test.js [hunk 1]
```diff
@@ -488,4 +492,23 @@ describe('conventionalcommits.org preset', function () {
...
+        issueUrlFormat: 'issues://{{repository}}/issues/{{prefix}}{{id}}',
+        issuePrefixes: ['#', 'GH-']
+      })
+    })
+      .on('error', function (err) {
+        done(err)
+      })
+      .pipe(through(function (chunk) {
+        chunk = chunk.toString()
+        expect(chunk).to.include('[#1](issues://conventional-changelog/issues/#1)')
```
[the remark this answers is not in the document]
Comment (tommywo) [reply packages/conventional-changelog-conventionalcommits/test/test.js]: this is made mainly to work with different issue trackers like JIRA, YOUTRACK where url format is like `https://jira.atlassian.com/browse/BSERV-10559`  `https://youtrack.jetbrains.com/issue/IDEA-217391`

[the remark this answers is not in the document]
Comment (tommywo) [reply packages/conventional-changelog-conventionalcommits/test/test.js]: with with jira like:
```
{
        issueUrlFormat: 'https://jira.atlassian.com/browse/{{prefix}}{{id}}',
        issuePrefixes: [ 'BSERV-']
}
```

[the remark this answers is not in the document]
Comment (tommywo) [reply packages/conventional-changelog-conventionalcommits/test/test.js]: i'v corrected the tests to address above examples.

**Comment by tapsboy** [comment]
Is there any example of how the JIRA issue format can be used? I tried putting the below in a JSON file /tmp/context.json
```
{
        issueUrlFormat: 'https://jira.atlassian.com/browse/{{prefix}}{{id}}',
        issuePrefixes: [ 'BSERV-']
}
``` 
and ran `npx conventional-changelog-cli -p conventionalcommits -c /tmp/context.json`
but still get github URLs instead of JIRA links

**Pull request merged**